keyboards/converter/numeric_keypad_IIe/config.h

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2019 Adam Newbold <adam@neatnik.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Apple Inc.
+#define PRODUCT         Numeric Keypad //e
+#define DESCRIPTION     Numeric Keypad //e, A2M2003
```

Comment:
I did say on Discord that commas aren't allowed in the USB Device Descriptor, but looking this over, I made a realization so obvious that I'm actually kind of embarrassed to admit to it now.

Wrap the value in quotation marks and you can use commas in the USB Device Descriptor.

```suggestion
#define DESCRIPTION     "Numeric Keypad IIe, A2M2003"
```

Made this change locally and it compiles with no errors.